src/main/java/org/fundacionjala/coding/erik/FizzBuzz.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.erik;
+
+import java.util.StringJoiner;
+
+/**
+ * this is class FizzBuzz.
+ */
+public class FizzBuzz {
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+
+    /**
+     * this is method for the numbers Fizz, Buzz or FizzBuzz.
+     *
+     * @param number is number limit.
+     * @return Numbers Fizz, Buzz or FizzBuzz.
+     */
+    public String fizzBuzz(int number) {
```

Comment:
You can split this method in two
one of them given one number will return fizz, buzz, or fizz buzz